cmd/state-installer/cmd.go

```diff
@@ -300,14 +312,6 @@ func execute(out output.Outputer, cfg *config.Instance, an analytics.Dispatcher,
 		return postInstallEvents(out, cfg, an, params, isUpdate)
 	}
 
-	// Check if state tool already installed
-	if !params.force && stateToolInstalled {
-		logging.Debug("Cancelling out because State Tool is already installed")
-		out.Print(fmt.Sprintf("State Tool Package Manager is already installed at [NOTICE]%s[/RESET]. To reinstall use the [ACTIONABLE]--force[/RESET] flag.", installPath))
-		an.Event(AnalyticsFunnelCat, "already-installed")
-		return postInstallEvents(out, cfg, an, params, true)
-	}
-
 	// If no sourcePath was provided then we still need to download the source files, and defer the actual
 	// installation to the installer contained within the source file
 	return installFromRemoteSource(out, cfg, an, args, params)
```

Comment:
We should get rid of this flow, since it's now replaced by the install.sh and install.ps1 and so there's no use-case for this anymore. Keeping it around probably makes our code more complex and error-prone than it needs to be.

Reply:
Okay.

Reply:
Oh sorry, GitHub confused me with both its e-mail and review interface. I thought something completely different :(